nucypher/blockchain/eth/agents.py

```diff
@@ -575,7 +575,10 @@ def release_worker(self, staker_address: ChecksumAddress) -> TxReceipt:
         return self.bond_worker(staker_address=staker_address, worker_address=NULL_ADDRESS)
 
     @contract_api(TRANSACTION)
-    def commit_to_next_period(self, worker_address: ChecksumAddress, fire_and_forget: bool = True) -> TxReceipt:  # TODO: make fire_and_forget required
+    def commit_to_next_period(self,
+                              worker_address: ChecksumAddress,
+                              fire_and_forget: bool = True,  # TODO: make fire_and_forget required? See #2385 too.
+                              ) -> TxReceipt:
```

Comment:
Should the typehint for this function be `Union[TxReceipt, HexBytes]` since it is transaction_hash or receipt?

Reply:
:heavy_check_mark: